src/feed-item.tsx

```diff
@@ -25,7 +26,7 @@ export class FeedItem extends React.Component<Properties> {
           </div>
           <CloudMedia
             className='feed-item__image'
-            src={imageUrl}
+            src={animationUrl || imageUrl}
```

Comment:
testsssss!!!!!! especially for the logic around which url it is prioritizing here and the feed leaf. thanks!